README.md

```diff
@@ -38,8 +38,6 @@ This is a pure PHP SDK for working with <b>Minter</b> blockhain
 
 ## Installing
 
-You need to ensure that you have installed <b>BCmath</b> extension for PHP.
```

Comment:
Removed, as there is no need to manually ensure for BCmath presence. Composer will do this for us.